src/env.ts

```diff
@@ -151,6 +152,9 @@ export function setEnvConfiguration(config: Configuration, handlers: FunctionInf
     if (config.enableDDTracing !== undefined && environment[ddTracingEnabledEnvVar] === undefined) {
       environment[ddTracingEnabledEnvVar] = config.enableDDTracing;
     }
+    if (config.enableXrayTracing !== undefined && environment[ddMergeXrayTracesEnvVar] === undefined) {
```

Comment:
Maybe you could extract this logic in a pure function, and add unit tests around it?

Reply:
Do you mean extract into a function like this?

```
function shouldUpdateEnvironment(configVariable: boolean | undefined, environmentVariable: undefined) {
  return configVariable !== undefined && environmentVariable === undefined;
}
```

I'm totally happy to extract it and write tests but I'm not sure I totally follow why that would be necessary since we already test the logic with the tests we currently have in `env.spec,ts` and this seems like a fairly trivial undefined check.